spec/Section 7 -- Response.md

```diff
@@ -8,19 +8,26 @@ request.
 A response may contain both a partial response as well as encountered errors in
 the case that a field error occurred on a field which was replaced with {null}.
 
+## Request Errors
+
+Any errors that occur during a GraphQL request are termed "request errors."
+Request errors are not limited to errors that occur during execution of the
+operation, they could be raised during parsing and validation of the request
+too. For example, a request error will be raised if the request does not
+contain sufficient information to indicate uniquely which operation to execute.
 
 ## Response Format
 
-A response to a GraphQL operation must be a map.
+A response to a GraphQL request must be a map.
 
-If the operation encountered any errors, the response map must contain an
+If the request encountered any errors, the response map must contain an
 entry with key `errors`. The value of this entry is described in the "Errors"
 section. If the operation completed without encountering any errors, this entry
 must not be present.
```

Comment:
Note the request might encounter an error before the operation can be determined (for example if `operationName` is not given but the document contains multiple operations, or of the document contains no operations), hence 'If the request encountered any errors.' However the final sentence is still correct - it does indeed relate to the **operation** completing without errors - for the operation to even begin there must be no errors before this point. Since it's valid both ways (and there's no ambiguity) I left this final sentence unedited.